resources/windows/wix/appdatafiles.wxs

```diff
@@ -0,0 +1,160 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <!--
+    Puppet Agent Specific Registry Entries
+    This overrides the generic file in the vanagon directory
+  -->
+  <Fragment>
+    <DirectoryRef Id="TARGETDIR">
+      <Directory Id="CommonAppDataFolder" Name="CommonAppData">
+        <Directory Id="APPDATADIR" Name="PuppetLabs">
+          <Directory Id="pxpAgentDataFolder" Name="pxp-agent">
+            <Directory Id="pxpAgentDataFolderVar" Name="var">
+              <Directory Id="pxpAgentDataFolderLog" Name="log" />
+              <Directory Id="pxpAgentDataFolderRun" Name="run" />
+              <Directory Id="pxpAgentDataFolderSpool" Name="spool" />
+            </Directory>
+            <Directory Id="pxpAgentDataFolderEtc" Name="etc">
+              <Directory Id="pxpAgentDataFolderModules" Name="modules"/>
+            </Directory>
+          </Directory>
+          <Directory Id="PuppetAppData" Name="puppet">
+            <Directory Id="PuppetConfDir" Name="etc"/>
+            <Directory Id="PuppetVarDir" Name="var"/>
+          </Directory>
+        </Directory>
+        <Directory Id="MCOAppData" Name="mcollective">
+          <Directory Id="MCOConfDir" Name="etc"/>
+          <Directory Id="MCOVarDir" Name="var">
+            <Directory Id="MCOLogDir" Name="log"/>
+          </Directory>
+        </Directory>
+        <Directory Id="FacterAppData" Name="facter">
+          <Directory Id="FactsDotD" Name="facts.d"/>
+        </Directory>
+        <Directory Id="CodeDir" Name="code">
+          <Directory Id="HieraDataDir" Name="hieradata"/>
+          <Directory Id="CodeModulesDir" Name="modules"/>
+          <Directory Id="EnvironmentsDir" Name="environments">
+            <Directory Id="ProductionDir" Name="production">
+              <Directory Id="ManifestsDir" Name="manifests"/>
+              <Directory Id="ModulesDir" Name="modules"/>
+            </Directory>
+          </Directory>
+        </Directory>
+      </Directory>
+    </DirectoryRef>
+    <ComponentGroup Id="AppDataComponentGroup" >
```

Comment:
Can you indent all the component lines as well - really anything with 3 or more items should be indented for clarity.